lib/widgets/tb_elevated_button.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:tech_byte/utils/colors.dart';
+import 'package:tech_byte/utils/constants.dart';
+
+enum TBButtonType { primary, secondary }
```

Comment:
In my opinion it is more descriptive as filled, outlined than primary and secondary.